### PR TITLE
feat: move console.debug to console.log in docs

### LINKED ISSUE #547: Move from console.debug to console.log on documentation examples
[FRO-74](https://linear.app/fuel-network/issue/FRO-74/move-from-consoledebug-to-consolelog-on-documentation-examples)

Recently we have moved from `console.log` to `console.debug` on the docs examples, but this causes confusion as the user needs to change the log level in order to see the logs.

We should move back all console.debug on the docs examples to console.log.

### PR DESCRIPTION
**Pull request by tibi77**
Closes #547

### CODE REVIEW

**Comment by tibi77** [comment]
I can not link issues, but this is related to #547

**Comment by tibi77** [comment]
closes #547

**Review by tomiiide — APPROVED** [review]
LGTM.

Nice work @tibi77

**Comment by tibi77** [comment]
@tomiiide I can not merge.. so if you d do the honor

**Comment by LuizAsFight** [comment]
@luizstacio need some eyes here this PR is approved but not able to be merged apparently because needs vercel 
 CI step completed... I tried to authorize but it's not moving anywhere
 
image

File: .changeset/honest-colts-tan.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'fuels-wallet': patch
+'@fuel-wallet/sdk': patch
+'@fuel-wallet/types': patch
+---
+
+Change console debug to console log in docs
```
Comment (luizstacio) [inline .changeset/honest-colts-tan.md:7]: As the changes is only on the docs you can remove this changeset file.

**Review by luizstacio — CHANGES_REQUESTED** [review]

**Review by luizstacio — APPROVED** [review]

**Pull request merged**